dracut/modules.d/90kiwi-live/kiwi-live-lib.sh

```diff
@@ -201,11 +201,17 @@ function preparePersistentOverlayDiskBoot {
     # used as presented
     local overlay_mount_point=$1
     local partitions_before_cow_part
+    local read_part_cmd
     mkdir -m 0755 -p "${overlay_mount_point}"
     if ! mount -L cow "${overlay_mount_point}"; then
         partitions_before_cow_part=$(_partition_count)
         echo -e "n\np\n\n\n\nw\nq" | fdisk "${isodiskdev}"
-        if ! partprobe "${isodiskdev}"; then
+        if type partprobe &> /dev/null;then
+            read_part_cmd="partprobe ${isodiskdev}"
+        else
+            read_part_cmd="partx -u ${isodiskdev}"
```

Comment:
I wonder why we use yet another way to re-read the partition table. In ```kiwi-partitions-lib.sh``` we use either partprobe or ```blockdev --rereadpt```. Both tools blockdev and partx belongs to util-linux. For consistency reasons I suggest that our fallback from partprobe is the same for 99kiwi-lib and 90kiwi-live. ```blockdev --rereadpt``` has proven to do the job, so maybe we stick to it, unless you have reasons that favors one over the other ?

The reasons why we have a bit of code duplication between 99kiwi-lib and 90kiwi-live is because we had planned to get rid of 90kiwi-live if all live ISO features went into dracut upstream. As long as we have the two code paths they should be consistent.

Thoughts ?